lsnms/nms.py

```diff
@@ -71,7 +71,7 @@ def _nms(
         keep.append(current_idx)
         to_consider[current_idx] = False
 
-    return np.array(keep)
+    return np.argwhere(score_mask)[keep]
```

Comment:
Numba can not index arrays (the result of `np.argwhere`) with a typed list (`keep`). To make the CI pass you just need:
```suggestion
    keep = np.array(keep)
    return np.argwhere(score_mask)[keep]
```